walmart/walmart.py

```diff
@@ -110,7 +109,12 @@ def send_request(
                             method, url, params, body, request_headers
                         )
                 raise
-        return response
+        try:
+            return response.json()
+        except ValueError:
+            # In case of reports, there is no JSON response, so return the
+            # content instead which contains the actual report
+            return response.content
```

Comment:
and which case is this?

Reply:
For reports, wrote a comment also for that